datafusion/core/tests/sqllogictests/test_files/type_coercion.slt

```diff
@@ -44,10 +44,10 @@ SELECT '2023-05-01 12:30:00'::timestamp - interval '1 month';
 
 # TODO: https://github.com/apache/arrow-datafusion/issues/6180
```

Comment:
I wonder if we can remove the link to the issue as it now works

Reply:
Removed